client/blocks/eligibility-warnings/index.jsx

```diff
@@ -29,32 +29,32 @@ function getHoldMessages( translate ) {
 		},
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
-			description: translate( 'This feature is only allowed on sites with business plan.' ),
+			description: translate( 'This feature is only allowed on sites with a business plan.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
 		NO_VIP_SITES: {
-			title: translate( 'VIP Site' ),
+			title: translate( 'VIP site' ),
 			description: translate( 'This feature is not supported on VIP sites.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
 		NO_WPCOM_NAMESERVERS: {
-			title: translate( 'No WordPress.com Name Servers' ),
+			title: translate( 'No WordPress.com name servers' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Use WordPress.com name servers' )` *ES Score: 11.85*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/eligibility-warnings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).